miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -676,6 +776,19 @@ export.get_account_id
     mem_load
 end
 
+#! Sets the account id and nonce.
+#!
+#! Stack: [account_nonce, 0, 0, account_id]
+#! Output: []
+#!
+#! Where:
+#! - account_id is the id of the currently accessing account.
+#! - account_nonce is the nonce of the currently accessing account.
+export.set_acct_id_and_nonce
+    exec.get_current_account_data_ptr push.ACCT_ID_AND_NONCE_OFFSET add
+    mem_storew
+end
```

Comment:
Is the `Output` correct here? `mem_storew` doesn't consume the elements from the stack, so this would leave the stack the same, or not?

Reply:
Good catch! Indeed, `Output` here is incorrect: it should be `[account_nonce, 0, 0, account_id]`. I'll update the comment.